scripts/buck_setup.sh

```diff
@@ -0,0 +1,27 @@
+cd third_party
+
+echo "\n[Generating wrappers for cpuionfo]"
```

Comment:
Doesn't this print out a literal `\n`?  I think you want to use printf instead of echo.  (This also requires you to put an explicit `\n` at the end.)

Reply:
it actually works but yeah I can change to printf